test/runners/javascript/modules/ethereum.js

```diff
@@ -0,0 +1,241 @@
+"use strict";
+
+const expect = require('chai').expect;
+
+const runner = require('../../../runner');
+
+describe('ethereum', function() {
```

Comment:
I'm glad I had split `javascript_spec.js`. Tests are much easier to find and write :)

Reply:
Yeah great idea! One thing I did notice is that the new files no longer have _spec as a suffix, and the naming scheme switched to kebob case. Was that on purpose?

Reply:
`_spec` seems redundant to me because these are under `test/`, but if you want them back I don't mind. kebab case file names are more common in JS projects and matches rest of the codebase.

Reply:
Ah, I can't keep them straight. I could have swore this repo was underscore cased, but apparently its just the specs. 

The useful thing about `_specs` is that when using file find its nice to see that a file is a spec (in case there are other files named the same), but also because there may be helper files within the directory that are not actually specs. Not a big deal though.